src/mds/MDSRank.cc

```diff
@@ -2650,9 +2652,95 @@ void MDSRankDispatcher::handle_asok_command(
       };
       l.lock();
     }
-    if (!op_tracker.dump_ops_in_flight(f, false, {""}, false, lambda)) {
-      *css << "op_tracker disabled; set mds_enable_op_tracker=true to enable";
+    if (!path.empty()) {
```

Comment:
When I saw this option defined in the command description, I assumed it would limit the ops being dumped to the path specified. Now I realize it is providing a path to a file for the output. I would appreciate a less confusing name for the option, given that `path` is also used to denote a file system path.

Reply:
"dump cache" also uses `path` so I used the same switch name for consistency.

Reply:
oh well. Then the better name is also wanted there, I guess :). Not a deal breaker for this PR.